addons/mail/static/src/components/chatter/chatter_suggested_recipient_tests.js

```diff
@@ -389,9 +389,8 @@ QUnit.test("suggested recipients should not be notified when posting an internal
     await this.start({
         async mockRPC(route, args) {
             if (args.model === 'res.fake' && args.method === 'message_post') {
-                assert.strictEqual(
-                    args.kwargs.partner_ids.length,
-                    0,
+                assert.notOk(
+                    args.kwargs.partner_ids,
```

Comment:
Why this change? I don't see where in the code we don't set `partner_ids` in `postData`. It seems to always be there.

Reply:
I don't remember... Guess it's some debug left over.